draft-jiang-sidrops-psvro.md

```diff
@@ -98,54 +98,54 @@ This document aims to provide insights and recommendations to network operators,
 
 According to measurements from {{IRRegularities}}, Route objects from JPIRR, RIPE, ARIN, ALTDB, and LACNIC have overlap rates in recent BGP announcements exceeding 60%, while databases such as APNIC, NTTCOM, WCGDB, PANIX, and ARIN-NA have rates below 20%. This indicates that some databases have lower activity and may contain a higher proportion of outdated data. The RADB database, which has the highest overlap rate, has 29.8% of its Route objects appearing in BGP announcements.
 
-## Lack of Support for MOAS
+## Support for Multi-origin ASes
 
 {{RFC1930}} suggests that a prefix should only have a single AS as its origin with a few exceptions. However, according to routing data from Routeviews{{CAIDA}}, MOAS (Multi-origin ASes) has become a common phenomenon, which can be due to multi-homing, Internet exchange points, and multinational companies, as well as due to misconfigurations or prefix hijacking.
 
 Analyzing the MOAS state, as of 2024, the currently active IRR databases can provide full coverage for only 31% of IPv4 MOAS and 4% of IPv6 MOAS. The RPKI database can provide full coverage for 26% of IPv4 MOAS and 3.55% of IPv6 MOAS. Differentiating between legitimate MOAS and those caused by misconfigurations or route hijacking poses a significant challenge for existing route origin databases.
 
-Legitimate MOAS is primarily caused by factors such as multinational companies and multi-homing. However, the current authoritative IRR and RPKI databases typically only allow registration of address blocks managed by the respective RIR. This poses a significant obstacle to support legitimate MOAS. On the other hand, misconfigurations usually occur within the same ISP, i.e., the ISP has multiple ASNs and IP prefixes, resulting from confusion during the configuration/update Route or ROA objects. These cases can bypass the current validation mechanisms.
+Legitimate MOAS is primarily caused by multinational companies and multi-homing. However, the current authoritative IRR and RPKI databases typically only allow registration of address blocks managed by the respective RIR. This poses a significant obstacle to support legitimate MOAS. On the other hand, misconfigurations usually occur within the same ISP, i.e., the ISP has multiple ASNs and IP prefixes, resulting from confusion during the configuration/update Route or ROA objects. These cases can bypass the current validation mechanisms.
 
 ## Inconsistency Among Multiple Sources
 
 As analyzed in section 2.1, relying on a single source of route origin database is insufficient for route origin validation. However, the IPv4 address coverage can reach 67% by integrating multiple active IRR databases. Further, the IPv4 address coverage can reach 70% with the addition of the RPKI database. Moreover, the number of ASes participating in IRR or RPKI accounted for 86.5% of the total number of assigned ASes.
```

Comment:
It is better to use {{xxx}} instead of section 2.1 directly and the reference label can be {#xxx}.